deploy/ironic_bmo_configmap.yaml

```diff
@@ -4,10 +4,12 @@ metadata:
   name: ironic-bmo-configmap
 data:
   http_port: "6180"
-  http_ip: "172.22.0.1"
-  interface: "ens3"
+  provisioning_interface: "ens3"
+  provisioning_ip: "172.22.0.3/24"
   dhcp_range: "172.22.0.10,172.22.0.100"
-  deploy_kernel_url: "http://172.22.0.1/images/ironic-python-agent.kernel"
-  deploy_ramdisk_url: "http://172.22.0.1/images/ironic-python-agent.initramfs"
-  ironic_endpoint: "http://localhost:6385/v1/"
-  ironic_inspector_endpoint: "http://localhost:5050/v1/"
+  deploy_kernel_url: "http://172.22.0.3:6180/images/ironic-python-agent.kernel"
+  deploy_ramdisk_url: "http://172.22.0.3:6180/images/ironic-python-agent.initramfs"
+  ironic_endpoint: "http://172.22.0.3:6385/v1/"
+  ironic_inspector_endpoint: "http://172.22.0.3:5050/v1/"
+  cache_url: "http://172.22.0.1/images"
+  rhcos_image_url: "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190520.0/"
```

Comment:
This is OpenShift specific, so I would prefer it didn't go anywhere in the baremteal-operator repository

Reply:
Also this image is now outdated, should be https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.2/420.8.20190624.0/

I guess that will be easier to maintain if we do move this yaml to dev-scripts, until the MAO integration lands (where I assume we can template this in based on knowledge of the release payload cc @sadasu )